### PR TITLE
127238-status-circle

### PR DESCRIPTION
**Pull request by boskya**
127238
### Review
 - [x] @hyperd 

### After Merge

run:

```
git checkout master
git pull
npm version patch
npm publish
```

### note

If npm version is `2.13` or later npm version will likely fail to run postversion script, leaving you with pending commits in your repo.

If this is the case, run `npm run-script postversion` to finish the process.

### CODE REVIEW

File: docs/site-styles/components/_styles.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.status-row {
+    display: flex;
+    margin-bottom: 5px;
+
+    .status.circle {
+        width: 12px;
```
Comment (hyperd) [inline docs/site-styles/components/_styles.scss]: Maybe these values should be moved to a _variables.scss file.

Comment (boskya) [inline docs/site-styles/components/_styles.scss]: moved it.

Comment (hyperd) [inline docs/site-styles/components/_styles.scss]: 👍

**Comment by hyperd** [comment]
:shipit:

**Pull request merged**